.buildkite/pipeline.yml

```diff
@@ -2,6 +2,9 @@ env:
   LC_ALL: "en_US.UTF-8"
   NIX_PATH: "channel:nixos-21.11"
 
+  # Avoid segmentation faults
+  GC_DONT_GC: 1
```

Comment:
Hmm... this comment has made me curious! How does this avoid segmentation faults?

Could we add a comment explaining why, and what happens if this line is removed?

Reply:
_Trying again without it..._